src/mmw/js/src/modeling/views.js

```diff
@@ -274,6 +274,7 @@ var ScenarioTabPanelView = Marionette.ItemView.extend({
 
     templateHelpers: function() {
         return {
+            active: this.model.get('active'),
```

Comment:
Do we need this? I thought you could reference `active`, like how we access `is_current_conditions` because the model is passed to the template.